alonzo/formal-spec/ledger.tex

```diff
@@ -2,12 +2,12 @@ \section{Ledger State Transition}
 \label{sec:ledger-trans}
 
 Figure~\ref{fig:rules:ledger} separates the case where all the scripts
-validate in a transaction from the case where there is one that does not.
-These cases are distinguished by the use of the $\IsValidating$ tag.
+in a transaction successfully validate from the case where there is one or more that does not.
+These two cases are distinguished by the use of the $\IsValidating$ tag.
```

Comment:
Can you make a note somewhere that, in the system as it currently is (and hopefully forever), it is always possible to *run* every script. However, this does not mean that there is a way to submit a transaction such that the script actually *validates*.

Reply:
Isn't this just the fact that the function that evaluates scripts is total? Also, the ability to run a script is technically irrelevant, we really only care whether it validates.

Reply:
I think the more interesting thing to say is that we expect that the `Script` type to only grow in future versions of the ledger. Is that what you meant?

Reply:
I did mean that the function is total. Definitely add what you said ^^ and maybe reiterate that the reason this function is total is that we intend to try to maintain the ability to translate any future new tx format to data expected by all non-native scripts that have been created in the past (but this cannot be a guarantee for until the end of time). Or something like that.